commands/azureCommands/quick-build.ts

```diff
@@ -1,94 +1,10 @@
-    const tarFilePath: string = getTempSourceArchivePath();
-    const client: ContainerRegistryManagementClient = await AzureUtilityManager.getInstance().getContainerRegistryManagementClient(subscription);
-
-    //Begin readying build
-    status.show();
-
-    const uploadedSourceLocation: string = await uploadSourceCode(client, registry.name, resourceGroupName, rootFolder, tarFilePath);
-    status.appendLine("Uploaded Source Code to " + tarFilePath);
-
-    const runRequest: DockerBuildRequest = {
-        type: 'DockerBuildRequest',
-        imageNames: [imageName],
-        isPushEnabled: true,
-        sourceLocation: uploadedSourceLocation,
-        platform: { os: osType },
-        dockerFilePath: dockerItem.relativeFilePath
-    };
-    status.appendLine("Set up Run Request");
-
-    const run: Run = await client.registries.scheduleRun(resourceGroupName, registry.name, runRequest);
-    status.appendLine("Scheduled Run " + run.runId);
-
-    await streamLogs(registry, run, status, client);
-    await fse.unlink(tarFilePath);
-}
-
-async function uploadSourceCode(client: ContainerRegistryManagementClient, registryName: string, resourceGroupName: string, rootFolder: vscode.WorkspaceFolder, tarFilePath: string): Promise<string> {
-    status.appendLine("   Sending source code to temp file");
-    let source: string = rootFolder.uri.fsPath;
-    let items = await fse.readdir(source);
-    items = items.filter(i => !(i in vcsIgnoreList));
-    // tslint:disable-next-line:no-unsafe-any
-    tar.c({ cwd: source }, items).pipe(fse.createWriteStream(tarFilePath));
-
-    status.appendLine("   Getting Build Source Upload Url ");
-    let sourceUploadLocation: SourceUploadDefinition = await client.registries.getBuildSourceUploadUrl(resourceGroupName, registryName);
-    let upload_url: string = sourceUploadLocation.uploadUrl;
-    let relative_path: string = sourceUploadLocation.relativePath;
-
-    status.appendLine("   Getting blob info from Upload Url ");
-    // Right now, accountName and endpointSuffix are unused, but will be used for streaming logs later.
-    let blobInfo: IBlobInfo = getBlobInfo(upload_url);
-    status.appendLine("   Creating Blob Service ");
-    let blob: BlobService = createBlobServiceWithSas(blobInfo.host, blobInfo.sasToken);
-    status.appendLine("   Creating Block Blob ");
-    blob.createBlockBlobFromLocalFile(blobInfo.containerName, blobInfo.blobName, tarFilePath, (): void => { });
-    return relative_path;
-}
-
-function getTempSourceArchivePath(): string {
-    /* tslint:disable-next-line:insecure-random */
-    let id: number = Math.floor(Math.random() * Math.pow(10, idPrecision));
-    status.appendLine("Setting up temp file with 'sourceArchive" + id + ".tar.gz' ");
-    let tarFilePath: string = url.resolve(os.tmpdir(), `sourceArchive${id}.tar.gz`);
-    return tarFilePath;
+    scheduleRunRequest(dockerFileUri, "DockerBuildRequest", actionContext);
```

Comment:
I know this file. It's almost nonexistent now. Let me know if you changed your mind and want me to move this function somewhere else.